src/main/java/ua/chernonog/onlinebookstore/dto/request/order/OrderRequestDto.java

```diff
@@ -0,0 +1,10 @@
+package ua.chernonog.onlinebookstore.dto.request.order;
+
+import jakarta.validation.constraints.NotBlank;
+import lombok.Data;
+
+@Data
+public class OrderRequestDto {
+    @NotBlank(message = "shippingAddress can't be  empty")
```

Comment:
As far as I understand the final message will be: shippingAddress shippingAddress can't be  empty. To avoid duplication you can write just: field can't be  empty

Reply:
![image](https://github.com/SkillfulDev/online-book-store/assets/73792778/e3e7272c-fd76-4484-b7ea-7865844a1239)
But I will change anyway